web/package.json

```diff
@@ -5,7 +5,8 @@
   "license": "MIT",
   "scripts": {
     "start": "webpack-dev-server --mode development --open",
-    "build": "webpack --mode production"
+    "build": "webpack --mode production",
+    "test": "jest"
```

Comment:
Jest dependency and its config it's missing. Let me know if you want to pair progamming on that or if it's okay I push a commit with Jest set up.